migrations/d7_media_theplatform_mpx_source_field_instance.yml

```diff
@@ -1,6 +1,11 @@
 id: d7_media_theplatform_mpx_source_field_instance
 label: Source field instance for mpx objects
 migration_group: media_theplatform_mpx
+migration_tags:
+  - 'Drupal 7'
```

Comment:
Can you add the Configuration tag? TugBoat runs these first at https://github.com/nbc-bravo/d8/blob/master/Makefile#L72.